frontend/src/components/ShootHibernation/HibernationConstraintWarning.vue

```diff
@@ -0,0 +1,49 @@
+<!--
+SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div v-if="!isHibernationPossible && hasHibernationSchedules" :class="{ 'd-flex' : icon }">
```

Comment:
The `v-if="!isHibernationPossible && hasHibernationSchedules"` should be outside of the component.

Reply:
Do we really need the `flexbox` container in case of an icon?

Reply:
done